aspnetcore/blazor/tutorials/movie-database-app/part-6.md

```diff
@@ -118,6 +131,36 @@ The result after searching on `road warrior`:
 
 Notice that the search box loses the search value ("`road warrior`") when the movies are filtered. If you want to preserve the searched value, add the `data-permanent` attribute:
 
+:::zone-end
+
+:::zone pivot="vsc"
+
+Type "`Road Warrior`" into the search box and select the **:::no-loc text="Search":::** button to filter the movies:
+
+![Mad Max movies before filtering in the movies Index page. The search field has the value 'Road Warrior'.](~/blazor/tutorials/movie-database-app/part-6/_static/form-filter-sqlite.png)
+
+The result after searching on `Road Warrior`:
+
+!['The Road Warrior' Mad Max movie filtered using a GET request via an HTML form action](~/blazor/tutorials/movie-database-app/part-6/_static/form-filter-result-sqlite.png)
+
+Notice that the search box loses the search value ("`Road Warrior`") when the movies are filtered. If you want to preserve the searched value, add the `data-permanent` attribute:
+
+:::zone-end
+
+:::zone pivot="cli"
+
+Type "`Road Warrior`" into the search box and select the **:::no-loc text="Search":::** button to filter the movies:
+
+![Mad Max movies before filtering in the movies Index page. The search field has the value 'Road Warrior'.](~/blazor/tutorials/movie-database-app/part-6/_static/form-filter-sqlite.png)
+
+The result after searching on `Road Warrior`:
+
+!['The Road Warrior' Mad Max movie filtered using a GET request via an HTML form action](~/blazor/tutorials/movie-database-app/part-6/_static/form-filter-result-sqlite.png)
+
+Notice that the search box loses the search value ("`Road Warrior`") when the movies are filtered. If you want to preserve the searched value, add the `data-permanent` attribute:
+
+:::zone-end
+
 ```diff
 - <form action="/movies" data-enhance>
 + <form action="/movies" data-enhance data-permanent>
```

Comment:
Why is this change needed?

Reply:
The search box loses its value when the form is submitted. The component isn't interactive at this stage of the tutorial.

Reply:
OK, but then we need to also explain what data-permanent does. I think we should handle this in a separate PR.

Reply:
Line 160 isn't enough?

Reply:
Ugh, I totally missed that when I scanned the text 🤦. Looks great! 😄